src/Types.ts

```diff
@@ -536,7 +536,7 @@ export interface StrategyEventResponse {
 
 export interface SignedStrategyResponse extends StrategyMetadata {
   createdAt: string
-  expiryTime: string
+  expiryTime?: string | ProcessError
```

Comment:
what is the scenario where we would need ProcessError for this? if there is no expected error condition, we shouldn't allow this to be anything other than a string

Also, if there is no expiryTime, maybe it would be better to return max_uint256 instead of undefined?

Reply:
ExpiryTime is a Date, but we have a block number in RequireBlockNotMined primitive.  If the provider fails finding currentBlock (rate limit, timeout)  y plan to return ProcessError. 
And the optional, is to return undefined if strategy doesn't expires. Another option could be to return **null**, but there is no field with null as possible value.

Reply:
makes sense, I forgot that expiryTime needs to be computed based on current block!